not-to-release/sources/newsgroup/groups.google.com_misc.consumers_a534e32067078b08_ENG_20060116_030800.xml.conllu

```diff
@@ -238,7 +238,7 @@
 14	.	.	PUNCT	.	_	13	punct	13:punct	_
 
 # sent_id = newsgroup-groups.google.com_misc.consumers_a534e32067078b08_ENG_20060116_030800-0012
-# text = 1.  Blowback in Iraq
```

Comment:
Was this on purpose? I guess it would be okay to remove multiple spaces but the multiple spaces are present in the original text.

Reply:
More or less by accident. There was an unbreakable space which is not mentioned in the SpacesAfter= key (MISC column). Probably I should have edited SpacesAfter.

Reply:
@dan-zeman do you have thoughts on this? should spaces be normalized?

Reply:
No strong position. Is the validator happy with this? Technically it should not complain because if MISC does not say "SpaceAfter=No", there can be one or more whitespace characters between two tokens. But I would not be surprised if the validator or some other script did not expect this and simply assumed that there is always just one chr(32). SpacesAfter is optional and many scripts probably do not look for it.

Preserving the original whitespace characters in the text attribute sounds good initially, but in fact it is not possible to preserve everything: there cannot be a line break, for instance.